flict/flictlib/format/common.py

```diff
@@ -25,3 +25,26 @@
         'question': {'markdown': 'Q---'},
     },
 }
+
+def extract_compatibilities(verification):
+    """
+    this method assumes 1 package with 1 dependency
+    """
+    compats = []
+    for package in verification['packages']:
+        dependency = package['dependencies'][0]
```

Comment:
I think it would be better to loop over `dependencies` here

Reply:
removed